src/old-tree.scm

```diff
@@ -24,6 +24,10 @@
 				(DefinedPredicate "Skip Interaction?")
 
 				(SequentialAnd
+					(DefinedPredicate "Did Someone New Speak?")
+					(DefinedPredicate "Request interaction with person who spoke"))
+
+				(SequentialAnd
```

Comment:
I'm pretty sure you don't need to make any updates to old-tree.scm -- this is the old behavior tree, and it has been made obsolete by openpsi.  Its still there as ... err, no good reason. Its handy for debugging and sanity checking.